src/overlays/actors/ovl_Eff_Stk/z_eff_stk.c

```diff
@@ -28,16 +28,87 @@ ActorInit Eff_Stk_InitVars = {
     (ActorFunc)EffStk_Draw,
 };
 
-#endif
+void func_80BF0DE0(EffStk* arg0, PlayState* arg1);
 
-extern UNK_TYPE D_06009F60;
+void EffStk_Init(Actor* thisx, PlayState* play) {
+    EffStk* this = (EffStk*)thisx;
+    Actor_SetScale(&this->actor, 0.2f);
+    this->actionFunc = func_80BF0DE0;
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Eff_Stk/EffStk_Init.s")
+void EffStk_Destroy(Actor* thisx, PlayState* play) {
+    EffStk* this = (EffStk*)thisx;
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Eff_Stk/EffStk_Destroy.s")
+void func_80BF0DE0(EffStk* arg0, PlayState* arg1) {
+    s16 temp_v0;
+    s16 temp_v0_2;
+    u16 temp_v1;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Eff_Stk/func_80BF0DE0.s")
+    if (Cutscene_IsCueInChannel(arg1, 0x200U) != 0) {
```

Comment:
Since this function returns a boolean there's no need for the zero comparison.
There's also an enum for the second parameter
```suggestion
    if (Cutscene_IsCueInChannel(arg1, CS_CMD_ACTOR_CUE_512)) {
```